torch_geometric/datasets/pandas_heterodata/__init__.py

```diff
@@ -0,0 +1,10 @@
+from .hetero_df import (nx_to_HeteroDF, heteroDF_split, heteroDF_merge,
```

Comment:
Is there a need to create a new sub-package?

Reply:
i think so because we introduced a new data structure with from/to conversion ,and the split and merge functions are working only with that new data structure (`HetreoDF`)